src/commander/commander_path_tab.c

```diff
@@ -48,14 +48,19 @@ static t_error_code	add_slash_if_needed(char **tab)
 	return (SUCCESS);
 }
 
+// TODO add test on a bad PATH var
 t_error_code	get_path_tab(char ***ret, char **env)
 {
 	t_error_code	err;
 	char			*path;
 	char			**path_tab;
 
 	path = env_getvar(env, PATH_KEY);
+	if (!path)
```

Comment:
Okey i told there was a crash if minishell was called with no env, but this solves it, nice!!!!